test-root/1.txt

```diff
@@ -0,0 +1 @@
+non-empty test file
```

Comment:
how are these files used. I'm not seeing them in the test case

Reply:
It was for my initial tests only, now it is removed.